test/TesterInternal/StreamingTests/SingleStreamTestRunner.cs

```diff
@@ -372,6 +374,54 @@ private void Heading(string testName)
             await StopProxies();
         }
 
+        public async Task StreamTest_22_TestImmutabilityDuringStreaming()
+        {
+            Heading("StreamTest_22_TestImmutabilityDuringStreaming");
+
+            IStreamingImmutabilityTestGrain itemProducer = GrainClient.GrainFactory.GetGrain<IStreamingImmutabilityTestGrain>(Guid.NewGuid());
+            string producerSilo = await itemProducer.GetSiloIdentifier();
+
+            // Setup consumers one in silo of item producer and the other one in another silo.
```

Comment:
I missed the [NonSerialized] part of the test the first pass.  Thanks for the  clarification. I think I now understand how the test works.